miniscope_io/models/stream.py

```diff
@@ -16,6 +16,7 @@
 class StreamBufferHeaderFormat(BufferHeaderFormat):
     pixel_count: Range
 
+ALLOWED_MODES: Tuple[str, ...] = ("DAQ", "RAW_RECORD", "RAW_REPLAY")
```

Comment:
this will be a general note i'll make throughout this, but we want to mock the OkDev class rather than modify this one :)